### ISSUE TITLE
should ignore files like .DS_Store in data/projects

### ISSUE THREAD

**Issue by xream** [node-ci/nci#26]
```
/Users/xream/Project/xream/nci-boilerplate/node_modules/.1.0.8@nci/app.js:8
        if (err) throw err;
                 ^

Error: ENOTDIR: not a directory, scandir '/Users/xream/Project/xream/nci-boilerplate/data/projects/.DS_Store'

```

**Comment by okv** [comment]
That's true, all hidden files should be omitted while loading projects.
I'll try to fix it shortly.

**Comment by okv** [comment]
System files filtered out during projects load.
Published nci@1.0.9
Thanks for pointing out that.
